frontend/src/static/js/v_authorship.js

```diff
@@ -5,6 +5,20 @@ const filesSortDict = {
   fileType: (file) => file.fileType,
 };
 
+const activeSortSectionDict = {
+  lineOfCode: (file) => `(${file.lineCount} lines)`,
+  path: (file) => file.path,
+  fileName: (file) => file.path.split(/[/]+/).pop(),
+  fileType: (file) => `(${file.fileType})`,
+};
+
+const pathSectionDict = {
+  lineOfCode: (file) => file.path,
```

Comment:
it doesnt quite make sense to set `lineOfCode` to `file.path`
perhaps we should do some change to the underlying structure?